supportedBackends/local/src/main/scala/cromwell/backend/impl/local/LocalBackend.scala

```diff
@@ -1,29 +1,227 @@
+      val workflowInputEntries = workflowDescriptor.inputs collect {
+        case(fqn, value) if splitFqn(fqn)._1 == call.fullyQualifiedName => splitFqn(fqn)._2 -> value
+      }
+
+      // Inputs coming from the "input" keyword in the workflow declaration. These need to be evaluated because they're expressions
+      val evaluatedInputMappings = call.inputMappings mapValues evaluate
+
+      TryUtils.sequenceMap(evaluatedInputMappings, "Job Input evaluation") flatMap { inputs =>
+        val localizedInputs = localizeInputs(jobPaths, runsOnDocker, fileSystems, inputs ++ workflowInputEntries)
+        call.task.instantiateCommand(localizedInputs, callEngineFunction, pathTransformFunction)
+      }
+    }
+
+    instantiatedCommand match {
+      case Success(command) =>
+        logger.info(s"`$command`")
+        writeScript(command, if (runsOnDocker) jobPaths.callDockerRoot else jobPaths.callRoot)
+        logger.info(s"command: $processArgs")
+        process = Option(processArgs.argv.run(ProcessLogger(stdoutWriter writeWithNewline, stderrTailed writeWithNewline)))
+        // This can't run on the context.dispatcher EC because it blocks incoming message processing (in particular abort)
+        // TODO use a new separate, backend scoped EC ?
+        waitAndPostProcess(scala.concurrent.ExecutionContext.global)
+      case Failure(ex) => Future.successful(BackendJobExecutionFailedResponse(jobDescriptor.key, ex))
+    }
+  }
 
   /**
     * Abort a running job.
     */
-  override def abortJob: Future[JobAbortResponse] = ???
+  override def abortJob: Future[JobAbortResponse] = {
+    Future {
+      process foreach { _.destroy() }
+      process = None
+      BackendJobExecutionAbortSucceededResponse(jobDescriptor.key)
+    } recover {
+      case e => BackendJobExecutionAbortFailedResponse(jobDescriptor.key, e)
+    }
+  }
+
+  /**
+    * Restart or resume a previously-started job.
+    */
+  override def recover: Future[BackendJobExecutionResponse] = execute
+
+  /**
+    * Writes the script file containing the user's command from the WDL as well
+    * as some extra shell code for monitoring jobs
+    */
+  private def writeScript(instantiatedCommand: String, containerRoot: Path) = {
+    jobPaths.script.write(
+      s"""#!/bin/sh
+          |cd $containerRoot
+          |$instantiatedCommand
+          |echo $$? > rc
+          |""".stripMargin)
+  }
+
+  /**
+    * --rm automatically deletes the container upon exit
+    * -v maps the host workflow executions directory to /root/<workflow id> on the container.
+    * -i makes the run interactive, required for the cat and <&0 shenanigans that follow.
+    */
+  private def buildDockerRunCommand(image: String): String = {
+    val dockerDir = jobPaths.callDockerRoot
+    s"docker run --rm -v ${jobPaths.callRoot.toAbsolutePath}:$dockerDir -i $image"
+  }
+
+  private def waitAndPostProcess(ec: ExecutionContext): Future[BackendJobExecutionResponse] = Future {
```

Comment:
If you mean the default implicit `ExecutionContext` (context.dispatcher), no, because it blocks message processing while this future is waiting for the process to finish.
Instead of overriding the ec with another one, I just passed it explicitly to the method to avoid confusion.